extensions/azurePublish/src/components/azureProvisionDialog.tsx

```diff
@@ -337,18 +346,22 @@ export const AzureProvisionDialog: React.FC = () => {
     setPage(page);
     switch (page) {
       case PageTypes.AddResources:
+        telemetryClient.track('ProvisionAddResources', {});
```

Comment:
Nit: If the properties are empty, you don't need to pass an empty object.
```suggestion
        telemetryClient.track('ProvisionAddResources');
```